### PR TITLE
Rpc Refactor

### PR DESCRIPTION
**Pull request by kumavis**
This should be tested before merging...

### CODE REVIEW

**Comment by kumavis** [comment]
first see #65

**Comment by kumavis** [comment]
eslint errors are not related to my changes

File: lib/index.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ App.prototype.start = function(cb) {
 
     //account manager doesn't do anything yet
     // self.accountMan = new AccountMan(detailsDB);
-    self.rpc = new EthRPC({app: self});
+    self.rpc = new EthRpc(app);
```
Comment (wanderer) [inline lib/index.js]: @kumavis where is `app` defined at?

**Comment by wanderer** [comment]
awesome! looks good @kumavis :)

**Pull request merged**